modules/ROOT/pages/analytics-event-api.adoc

```diff
@@ -209,7 +235,12 @@ Your report can query data for one, many, or all of the available data fields. T
 |Violated Policy Name |The name of the policy violated by the API request (if any).
 |===
 
-If the Content-Length header is present, the Response Size is set to that value. If the Content-Length header is not present and the payload is a String, Analytics calculates the length of the String and reports that value. If the Content-Length header is not present and the payload is not a String, Analytics reports the response size as -1. For example, if the output returned is a DataWeave stream and the Content-Length header is not present, Analytics doesn't report a response size because the value is not a String. However if your application performs a String conversion, the response size is listed.
+In your query:
+
+* If the `Content-Length` header is present, the `Response Size` is set to that value. If the `Content-Length` header is not present and the payload is a String, Analytics calculates the length of the String and reports that value. 
```

Comment:
```suggestion
* If so, the `Response Size` is set to that value. 
* If not, and the payload is a String, Analytics calculates the length of the String and reports that value. 
```

Reply:
Made some formatting changes on my own that makes it easier to read now.